components/dashboard/src/components/Button.tsx

```diff
@@ -40,50 +43,59 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
-                              "bg-gray-100 dark:bg-gray-700 hover:bg-gray-200 dark:hover:bg-gray-600",
-                              "text-gray-500 dark:text-gray-100 hover:text-gray-600",
-                          ]
-                        : null,
-                    type === "danger" ? ["bg-red-600 hover:bg-red-700", "text-gray-100 dark:text-red-100"] : null,
-                    type === "danger.secondary"
-                        ? [
-                              "bg-red-50 dark:bg-red-300 hover:bg-red-100 dark:hover:bg-red-200",
-                              "text-red-600 hover:text-red-700",
-                          ]
-                        : null,
-                    {
-                        "w-full": size === "block",
-                        "cursor-default opacity-50 pointer-events-none": disabled || loading,
-                    },
-                    className,
-                )}
-                ref={ref}
-                disabled={disabled}
-                autoFocus={autoFocus}
-                onClick={onClick}
-            >
+        const commonProps = {
+            className: classNames(
+                "cursor-pointer px-4 py-2 my-auto",
+                "text-sm font-medium whitespace-nowrap",
+                "rounded-md focus:outline-none focus:ring transition ease-in-out",
+                type === "primary"
+                    ? [
+                          "bg-green-600 dark:bg-green-700 hover:bg-green-700 dark:hover:bg-green-600",
+                          "text-gray-100 dark:text-green-100",
+                      ]
+                    : null,
+                type === "secondary"
+                    ? [
+                          "bg-gray-100 dark:bg-gray-700 hover:bg-gray-200 dark:hover:bg-gray-600",
+                          "text-gray-500 dark:text-gray-100 hover:text-gray-600",
+                      ]
+                    : null,
+                type === "danger" ? ["bg-red-600 hover:bg-red-700", "text-gray-100 dark:text-red-100"] : null,
+                type === "danger.secondary"
+                    ? [
+                          "bg-red-50 dark:bg-red-300 hover:bg-red-100 dark:hover:bg-red-200",
+                          "text-red-600 hover:text-red-700",
+                      ]
+                    : null,
+                {
+                    "w-full": size === "block",
+                    "cursor-default opacity-50 pointer-events-none": disabled || loading,
+                },
+                className,
+            ),
+            children: (
                 <ButtonContent loading={loading} icon={icon}>
                     {children}
                 </ButtonContent>
-            </button>
-        );
+            ),
+        };
+
+        if (href) {
+            return <Link to={href} {...commonProps} />;
+        } else {
+            return (
+                <button
+                    type={htmlType}
+                    ref={ref as ForwardedRef<HTMLButtonElement>}
```

Comment:
do you want to handle ref being an anchor element differently?

Reply:
Nope, this was an oversight and I've now put it on the link as well...